x-pack/plugins/maps/public/routing/page_elements/top_nav_menu/index.js

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
Should `top_nav_menu` be moved to `connected_components` folder?

Reply:
There's a lot of shuffling that could and likely will happen. For example what was previously the `angular/` folder was renamed to `bootstrap/` in this PR. Both names are catch-all-y, and `bootstrap` doesn't even serve it's original purpose of consolidating like-written code, so code in this folder should likely eventually move. Likewise, for this PR I'd like to keep this routing-related logic together although I agree that in the long run it will also likely move.

To address your above comment:
> Maybe rename to MapsApp

I like Maps App for this route, I'll update it!